.devcontainer/devcontainer.json

```diff
@@ -3,7 +3,7 @@
 {
   "name": "NxDevContainer",
   // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-  "image": "mcr.microsoft.com/devcontainers/typescript-node:0-18",
```

Comment:
nitpick, let's not have this change in this PR, also probs shouldn't be latest as could end up using version that isn't fully supported yet. 

Mainly just doing want these changes to end up in PRs that aren't fully related. Sorry didn't catch this initially.

Reply:
Not changing it now, but my thought would be to change it to `0-20`

Changes made.